tools/runtime-stats-monitor.md

```diff
@@ -0,0 +1,65 @@
+---
+id: RNSM
+title: Runtime Network Stats Monitor
+sidebar_label: Runtime Network Stats Monitor
+---
+
```

Comment:
@ian-m-unity @s-omeilia-unity it would be great to have a section about custom data in the RNSM - that is one of the few features that is not accessible via the UI. if possible, a small code example?

Reply:
And one other suggestion, creating preset configurations using scriptable objects